Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-ruby "2.4.1"
+ruby RUBY_VERSION
```

Comment:
Beware this trick when deploying. I'm pretty sure it works with `heroku`, but I'm not 100% sure!